manpages/fsck.fat.8.in

```diff
@@ -66,6 +66,9 @@ The start pointer is adjusted.
 Directory .. does not point to parent of parent directory.
 The start pointer is adjusted.
 .IP "\(bu" 4
+. and .. are not the two first entries in a non\-root directory.
```

Comment:
`non-root` without backslash
```suggestion
. and .. are not the two first entries in a non-root directory.
```

Reply:
@fstirlitz: Would you fix this issue and rebase patch? Or should I do it?

Reply:
Did that already, just held out pushing in case I noticed something else.

Reply:
Now I see that you have pushed update, but it is not rebased on top of master branch.